src/App.tsx

```diff
@@ -10,33 +11,21 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {selectedTabId.title}
```

Comment:
You can use a string interpolation instead of three separate lines. It should looks better